backend/app.py

```
@@ -214,6 +254,76 @@ def analyze():
+                        counts["criticalFindings"] = len([f for f in updated_findings if f.get("severity") == "CRITICAL"])
+                        
+                        # Calculate a new threat score based on highest risk finding
+                        threat_score = case.get("threatScore", 0)
+                        if new_findings:
+                            highest_risk = max([f.get("riskScore", 0) for f in new_findings])
+                            if highest_risk > threat_score:
+                                threat_score = highest_risk
+                        
+                        # Determine severity based on threat score
+                        severity = "LOW"
+                        if threat_score >= 75: severity = "CRITICAL"
```

Ruff (0.16.2)

[error] Multiple statements on one line (colon)

(E701)